projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cpq-quote/cpq-quote.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function DiscountPercentageQuote() {
+  log('Discount Percentage Heading', DiscountPercentageQuote.name);
+  cy.get('#cx-item-list-discount')
+    .should('contain.text', 'Item Price')
+    .and('be.visible');
+}
+
+export function DiscountPercentageQuoterow() {
+  log('Discount Percentage Row', DiscountPercentageQuoterow.name);
+
+  cy.get('cx-cart-item-list') // Locate the parent element containing the rows
+    .find('tr[cx-cart-item-list-row]')
+    .find('cx-cpq-quote')
+    .first()
+    .should('be.visible');
+  cy.get('.cx-discount').should('be.visible');
+  cy.get('.cx-formattedValue').should('be.visible').should('not.have.text', '');
+}
+export function DiscountQuoterow() {
```

Comment:
Please, rename this method and add JS doc because it is not clear what the method does.